src/Command/ConfigDump/Generate.php

```diff
@@ -72,21 +72,21 @@ class Generate
      * @param File $file
      * @param ArrayManager $arrayManager
      * @param MagentoVersion $magentoVersion
-     * @param SharedConfig $sharedConfig
+     * @param Resolver $sharedConfig
      * @param PhpFormatter $phpFormatter
      */
     public function __construct(
         ConnectionInterface $connection,
         File $file,
         ArrayManager $arrayManager,
         MagentoVersion $magentoVersion,
-        SharedConfig $sharedConfig,
+        Resolver $sharedConfig,
```

Comment:
Please rename variable `$sharedConfig` as now it's Resolver

Reply:
It's the shared config resolver though, right? I don't disagree, just a thought.

Reply:
Renamed